.github/workflows/test.yml

```diff
@@ -50,33 +77,34 @@ jobs:
                   node-version: ${{ matrix.build-version }}
             - run: npm ci
             - run: npm run build
-            - run: npm run lint
-            - run: npm run format:check
             - name: Test using Node.js ${{ matrix.node-version }}
               uses: actions/setup-node@v4
               with:
                   node-version: ${{ matrix.node-version }}
             - run: npm i -g npm@8
             - run: npm test
-            - run: npm run smoketest
+            - run: npm run smoketest:node
               env:
                   SMOKETESTS_SUBMIT_LAYER_URL: ${{ secrets.SMOKETESTS_SUBMIT_LAYER_URL }}
                   SMOKETESTS_DIRECT_SUBMIT_URL: ${{ secrets.SMOKETESTS_DIRECT_SUBMIT_URL }}
 
     test_windows:
         runs-on: windows-latest
+
+        strategy:
+            matrix:
+                node-version: [18.x]
```

Comment:
do we need this matrix if we test only against 18?

Reply:
I think it's plain and simple to see then on which Node versions it is tested.